todo_app/templates/index.html

```diff
@@ -12,7 +12,20 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{ item.id }} - {{ item.status }} - {{ item.title }}</li>
```

Comment:
Really minor, but we could now get rid of the comment above this, and it would be nice to indent these list items

Reply:
Done, thanks.